app/app.go

```diff
@@ -464,7 +464,7 @@ func NewEvmos(
 
 	app.FeesKeeper = feeskeeper.NewKeeper(
 		keys[feestypes.StoreKey], appCodec, app.GetSubspace(feestypes.ModuleName),
-		app.AccountKeeper, app.BankKeeper, app.EvmKeeper,
```

Comment:
What's the difference btw AccountKeeper and BankKeeper?

Reply:
In this module, `AccountKeeper` is not used. It is an artifact from an initial version of the fees module. `BankKeeper` is used here, to send the fees shares to the contract deployer: https://github.com/tharsis/evmos/blob/04196b05a84967fb1597b64cfc3c17d38107d1a4/x/fees/keeper/evm_hooks.go#L65